Makefile

```diff
@@ -8,12 +8,25 @@ doc:
 	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
 	@echo
 
+doc3:
+	python3 setup.py build_sphinx
+	@echo
+	@echo Generated documentation: "file://"$$(readlink -f doc/build/html/index.html)
+	@echo
+
 test:
 	-find coverage/ -mindepth 1 -delete
 	python $$(which nosetests) $${TESTS}
 
+test3:
+	-find coverage/ -mindepth 1 -delete
+	python3 $$(which nosetests3) $${TESTS}
+
```

Comment:
this target should go. a separate `tox` environment is the preferred way to test against multiple python versions.

these makefile targets only exist because it makes development from within a virtualenv easier (no need to run `tox` which is slower).